Nim/Program.cs

```diff
@@ -0,0 +1,441 @@
+﻿using System.Text;
+
+namespace NimConsoleApplication
+{
+    public static class Program
+    {
+        private static int _winCountBasicMenu; private static int _lostCountBasicMenu;
+        private static int _winCountFunMenu; private static int _lostCountFunMenu;
+
+        static void Main()
+        {
+            bool exitProgram = false;
+            while (!exitProgram) {
+                string mainMenu = @$"                                     
+  /\/\    __ _ (#) _ __     /\/\    ___  _ __   _   _ 
+ /    \  / _` || || '_ \   /    \  / _ \| '_ \ | | | |
+/ /\/\ \| (_| || || | | | / /\/\ \| -__/| | | || |_| |
+\/    \/ \__,_||_||_| |_| \/    \/ \___||_| |_| \__,_|
+               __________________________
+                Please select an option
+               ¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨
+           <<  | [A] - Nim Game (basic) |  >>
+               | Wins: {_winCountBasicMenu}     Loses: {_lostCountBasicMenu}   |
+               ¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨
+           <<  | [B] - Nim Game (fun)   |  >>
+               | Wins: {_winCountFunMenu}     Loses: {_lostCountFunMenu}   |
+               ¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨
+           <<  | [Q] - Quit Program     |  >>
+               ¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨";
+
+                //I want to apply colour to sections of the mainMenu string, without writing multiple Console.WriteLine statements.
+                string[] menuLines = mainMenu.Split(Environment.NewLine); //TIL about C# Lines.
+                foreach (string menuLine in menuLines) {
+                    if (menuLine.Contains("Please select an option")) {
+                        Console.ForegroundColor = ConsoleColor.Magenta;
+                        
+                    } else if (menuLine.Contains("<<  | [A] - Nim Game (basic) |  >>")) {
+                        Console.ForegroundColor = ConsoleColor.Yellow;
+                        
+                    } else if (menuLine.Contains("<<  | [B] - Nim Game (fun)   |  >>")) {
+                        Console.ForegroundColor = ConsoleColor.Green;
+
+                    } else if (menuLine.Contains("¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨") ||
+                               menuLine.Contains("__________________________") || 
+                               menuLine.Contains($"| Wins: {_winCountBasicMenu}     Loses: {_lostCountBasicMenu}   |") ||
+                               menuLine.Contains($"| Wins: {_winCountFunMenu}     Loses: {_lostCountFunMenu}   |")) 
+                    {
+                        Console.ResetColor();
+                    } else {   
+                        Console.ForegroundColor = ConsoleColor.Red; // ASCII ART
+                    }
+                    Console.WriteLine(menuLine); //Print the newly coloured Menu.
+                } 
+                Console.ResetColor();
```

Comment:
This is quite hard to debug and has high complexity. Not that it'd be wrong, though.
It might be worth inventing your own protocol:

```c#
string message = @"[Color]Red
Something
[Color]Blue
Something";

string[] menuLines = mainMenu.Split(Environment.NewLine); //TIL about C# Lines.
                foreach (string menuLine in menuLines) {
                       if(menuLine.StartsWith("[Color]")){
                            // I'm sure you'll figure out the rest ;)
                       }
```

That way, you have less tight coupling of contents of the message and the color logic.